rebar.config

```diff
@@ -3,6 +3,7 @@
           , "deps"
           ]}.
 {erl_opts, [ {platform_define, "^R[0-9]+", erlang_deprecated_types}
+           , {platform_define, "^(R[0-9]+|17|18|19|20)", http_uri_depricated_functions}
```

Comment:
Starting with OTP 21, there's a OTP_RELEASE macro that you can use.
See https://erlang.org/doc/reference_manual/macros.html